### DISCUSSION TITLE
How to create subtitles for tiktok?

### DISCUSSION THREAD

**Discussion by orionflame** [openai/whisper#1647] [Q&A]
Hi,

Basically I have an speech audio (mp3) that I want to convert to an srt subtitle file so that I can render this text in Davinci software. I can do this in code no problem but the issue is the time window of the subtitles is too large. I want shorter time frame so instead of having 1 text for each 5 to 10 seconds, I want to have subtitles for each 1-2 seconds, so the text doesn't have to be so long on screen, but rather short and quick phrases.

Is this possible?

Thanks a lot in advance.

**Comment by glangford** [comment]
`--max_line_width` is your best option at the moment.

**Comment by dgoryeo** [comment]
Tiktok style is a bit tricky. Perhaps a bit too obvious solution: [CapCut](https://www.capcut.com) was designed for that. 

In Whisper ecosystem, [Stable_TS](https://github.com/jianfch/stable-ts) has various technics to group and break words/lines. May be that can do the trick.
